pandas_datareader/base.py

```diff
@@ -42,13 +42,14 @@ class _BaseReader(object):
     _chunk_size = 1024 * 1024
     _format = 'string'
 
-    def __init__(self, symbols, start=None, end=None,
+    def __init__(self, symbols, start=None, end=None, freq=None,
```

Comment:
No need to care values which is not implemented ATM from user's point of view. Also, pls move `freq` to `WorldBankReader` as other readers don't use it.

NOTE: Some other readers have `interval` kw which also accept frequency-like. It may better to use the same word but not sure as I'm not native English speaker.